.github/workflows/alpine.yaml

```diff
@@ -8,21 +8,54 @@ on:
       - "alpine/**"
       - ".github/workflows/alpine.yaml"
       - "install_tooling.sh"
+  workflow_call:
+    secrets:
+      IMAGE_PREFIX:
+        description: "registry.com/loftsh in registry.com/loftsh/language:TAG"
+        required: true
 
 env:
-  IMAGE_PREFIX: loftsh
+  IMAGE_PREFIX: ${{ secrets.IMAGE_PREFIX }}
+  LANGUAGE: "alpine"
+  SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
+
+defaults:
+  run:
+    working-directory: alpine
 
 jobs:
   alpine:
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - tag: edge
+            target_tag: edge
+          - tag: latest
+            target_tag: latest
+          - tag: 3.14
+            target_tag: 3.14
+          - tag: 3.13
+            target_tag: 3.13
+          - tag: 3.12
+            target_tag: 3.12
     steps:
       - uses: actions/checkout@v2
       - uses: docker/setup-qemu-action@v1
       - uses: docker/setup-buildx-action@v1
       - run: echo ${{ secrets.DOCKER_PASSWORD }} | docker login -u ${{ secrets.DOCKER_USERNAME }} --password-stdin
-      - run: chmod +x install_tooling.sh && sudo ./install_tooling.sh
-      - run: |
-          devspace build --var TAG=latest
-          devspace build --var TAG=3
-        working-directory: ./alpine
-
+      - uses: loft-sh/setup-devspace@main
+      - run: devspace build --var TAG=${{ matrix.tag }} --var TARGET_TAG=${{ matrix.target_tag }}
+      - name: Synk Docker scan for TAG=${{ matrix.target_tag }}
+        continue-on-error: true
+        uses: snyk/actions/docker@master
+        with:
+          image: ${{ env.IMAGE_PREFIX }}/${{ env.LANGUAGE }}:${{ matrix.target_tag }}
+          args: --file=go/Dockerfile --severity-threshold=high --exclude-base-image-vulns
+      - name: Upload result to GitHub Code Scanning
+        continue-on-error: true
+        uses: github/codeql-action/upload-sarif@v1
+        with:
+          sarif_file: snyk.sarif
+          category: "${{ env.IMAGE_PREFIX }}/${{ env.LANGUAGE }}:${{ matrix.target_tag }}"
```

Comment:
I think we can remove everything related to snyk here because we already have a paid snyk subscription hooked up to our docker hub account, so this is already done and does not need to be repeated in github actions

Reply:
thanks for the heads-up, removed